src/components/withHeroChart.js

```diff
@@ -94,13 +97,13 @@ const withHeroChart = (ComposedComponent) => {
           },
         },
         tooltip: {
-          enabled: true, //false,
+          enabled: true,
           shared: true,
           crosshairs: true,
           borderRadius: 8,
-          positioner: function(labelWidth, labelHeight, point) {
-            return { x: point.plotX, y: this.chart.plotTop + this.chart.plotHeight - labelHeight };
-          },
+          // positioner: function(labelWidth, labelHeight, point) {
```

Comment:
@micapam removing this makes the tooltip automagically place itself and addresses "
tooltips positioned to the left rather than on top of the line (or moves away from live line charts)" from https://govausites.atlassian.net/browse/PD-1402. 

Are you ok with this change?